src/views/Player.vue

```diff
@@ -228,7 +226,7 @@ export default defineComponent({
     /**
      * remove the scorecard, display last question and remove the confetti
      */
-    function toLastQuestion() {
+    function goToLastQuestion() {
       state.isScorecardShown = false;
       state.currentQuestionIndex -= 1;
       state.numCorrect = 0;
```

Comment:
why would we reset these metrics?

Reply:
Because if someone clicks the back button and then the child again goes to scorecard page , the whole metrics was being recalculated with the already set parameter such as numCorrect , so in this case, we needed to reset the parameters.